_events/2022-All-Hands-Meeting.md

```diff
@@ -3,7 +3,7 @@ short_title: OSG All-Hands Meeting 2022
 title: "OSG All-Hands Meeting 2022"
 published: true
 excerpt: |
-   Save the date for the OSG All-Hands Meeting 2022 to be held March 14-18 at the University of Oklahoma (OU) in Norman and virtually. We are planning a hybrid event. If you register for and travel to the University of Oklahoma (OU) in Norman for the in-person components, you can mingle with colleagues, have planned or ad hoc meetings, attend workshops or training events, and join hack-a-thons. In addition, everyone can enjoy the same kind of presentations and discussions as in the past 2 years, whether in-person or online. 
+   Registration is now open for the OSG All-Hands Meeting 2022 to be held March 14-18 at the University of Oklahoma (OU) in Norman and virtually. We are planning a hybrid event. If you register for and travel to the University of Oklahoma (OU) in Norman for the in-person components, you can mingle with colleagues, have planned or ad hoc meetings, attend workshops or training events, and join hack-a-thons. In addition, everyone can enjoy the same kind of presentations and discussions as in the past 2 years, whether in-person or online. 
```

Comment:
The second “the University of Oklahoma (OU) in Norman” is repetitive. I suggest changing the whole string to “OU”, since it is defined in the preceding sentence.

Reply:
Has anyone talked to Brian L. lately about running hackathons with only 1.5 hours available each afternoon? I am increasingly concerned that we are doing false advertising about that particular kind of event.